src/Compilers/Core/Portable/Emit/EditAndContinue/DeltaMetadataWriter.cs

```diff
@@ -541,6 +541,8 @@ protected override void CreateIndicesForNonTypeMembers(ITypeDefinition typeDef)
 
             // First we find the deleted methods, and add them to our dictionary. This is used later when
             // processing events, properties, methods, and references.
+            ImmutableDictionary<IMethodDefinition, DeletedMethodDefinition>? deletedMethodDefinitions = null;
```

Comment:
>null [](http://example.com/codeflow?start=104&length=4)

Perhaps this could be empty by default to avoid null checks and asserts later on